lib/backend/k8s/resources/node.go

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) 2016 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resources
+
+import (
+	log "github.com/Sirupsen/logrus"
+
+	"github.com/projectcalico/libcalico-go/lib/backend/api"
+ 	"github.com/projectcalico/libcalico-go/lib/backend/model"
+	"github.com/projectcalico/libcalico-go/lib/errors"
+
+	"k8s.io/client-go/kubernetes"
+	kapiv1 "k8s.io/client-go/pkg/api/v1"
+	metav1 "k8s.io/client-go/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+)
+
+func NewNodeClient(c *kubernetes.Clientset, r *rest.RESTClient) api.Client {
+	return &nodeClient{
+		clientSet: c,
+		tprClient: r,
+	}
+}
+
+// Implements the api.Client interface for Nodes.
+type nodeClient struct{
+	clientSet *kubernetes.Clientset
+	tprClient *rest.RESTClient
```

Comment:
No need for the tprClient here.  It's not used for nodes since they're not ThirdPartyResources.